aws/istio-ingress/base/ingress.yaml

```diff
@@ -1,16 +1,16 @@
-apiVersion: extensions/v1beta1 # networking.k8s.io/v1beta1
+apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   annotations:
-    kubernetes.io/ingress.class: alb
-    alb.ingress.kubernetes.io/scheme: $(loadBalancerScheme)
     alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}]'
+    alb.ingress.kubernetes.io/scheme: internet-facing
```

Comment:
This should be $(loadBalancerScheme). Could you revert changes on line 6-7? Don't have to change the sequence

Reply:
Resolved in the latest commit